core/assets/bundles/bundle.properties

```diff
@@ -660,6 +660,7 @@ keybind.press.axis = Press an axis or key...
 keybind.screenshot.name = Map Screenshot
 keybind.move_x.name = Move x
 keybind.move_y.name = Move y
+keybind.mouse_move.name = Mouse Move
```

Comment:
"Follow Mouse" would be more appropriate.

Reply:
Done.